doc/api/buffer.md

```diff
@@ -5137,9 +5137,13 @@ added: REPLACEME
 -->
 
 * input {Buffer | ArrayBuffer | TypedArray} The input to validate.
-* Returns: {boolean} Returns `true` if and only if the input is valid UTF-8.
+* Returns: {boolean}
+
+This function is used to check if input contains invalid UTF-8 code points
+(characters). Returns `true` if and only if the input does not contain
+non UTF-8 characters.
```

Comment:
It does seem a bit odd to define this via double negation... maybe that could be simplified, and if the desire is to clarify the behavior for an empty string (which is absolutely correct here, to be clear!), that could also just be mentioned explicitly?

```suggestion
This function returns `true` if `input` contains only valid UTF-8-encoded data,
including the case in which `input` is empty.
```

(UTF-8 code points aren't really a thing, UTF-8 refers to a specific way of encoding code points into byte sequences, so the result is a byte sequence that can either be valid or invalid as UTF-8)

Reply:
I missed this! I'm sorry for not getting back to you sooner. That makes sense. Updating the pull request.